lua/lvim/core/luasnip/init.lua

```diff
@@ -0,0 +1,59 @@
+local M = {}
+function M.config()
+  lvim.builtin.luasnip = {
+    sources = {
+      friendly_snippets = true,
+      lunarvim = true,
```

Comment:
```suggestion
      user = {
      	paths = {
      		utils.join_paths(get_config_dir(), "snippets")
      	}
      },
```

Reply:
My intention was to add lunarvim specific snippets, hence the name. It is a boolean to easily disable it in case you don't want them, it was not meant to be extended.
Allowing for arbitrary user paths is not trivial because luasnip needs the correct paths when you call the loader methods, you can't just call it and let it figure out which paths are for what things.

Reply:
okay, now I see what you mean! the problem is that we should still expose the snippets path (probably both `lua` and `json` as per https://github.com/LunarVim/LunarVim/pull/2962#discussion_r958185153)